org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/AboutWindow.java

```diff
@@ -367,7 +367,7 @@ protected Tabpanel createInfo() {
 		LayoutUtils.addSclass("about-info-panel", div);
 		div.setParent(tabPanel);
 		ZKUpdateUtil.setHeight(div, "100%");
-		div.setStyle("overflow: auto;");
+		div.setStyle("overflow: auto; line-height: 1.3em; font-size: 12px");

```

Comment:
I think this setStyle call should be remove. The settings should goes into the "about-info-panel" css class above instead.